### PR TITLE
Add models from mdr_hri_behaviors

### PR DESCRIPTION
**Pull request by ErickKramer**
Added mdr_hri_behaviors nlu models to the rasa_nlu_models package

### CODE REVIEW

File: rasa_nlu_models/common/mdr_hri_behaviours_models/nlu/checkpoint [hunk 1]
```diff
@@ -0,0 +1,2 @@
+model_checkpoint_path: "/home/patrick/rasa/models/current/nlu/intent_classifier_tensorflow_embedding.ckpt"
```
Comment (argenos) [inline rasa_nlu_models/common/mdr_hri_behaviours_models/nlu/checkpoint]: Is this file necessary?

Comment (ErickKramer) [reply rasa_nlu_models/common/mdr_hri_behaviours_models/nlu/checkpoint]: Most likely no. As I mentioned before, it is automatically generated when training the model. Should I remove them?. @argenos

**Comment by argenos** [comment]
Yes, please. Only include the relevant files.

**Review by argenos — CHANGES_REQUESTED** [review]
Please remove unnecessary files from the PR.

**Comment by argenos** [comment]
This PR is related to https://github.com/b-it-bots/mas_domestic_robotics/pull/194

**Review by minhnh — APPROVED** [review]

**Review by argenos — APPROVED** [review]
